src/utils/thread_helper.hpp

```diff
@@ -258,7 +258,9 @@ class ThreadHelper
 
 #if OTBR_ENABLE_TELEMETRY_DATA_API
     /**
-     * This method populates the telemetry data and returns the error code if error happens.
+     * This method populates the telemetry data with best effort and returns the error code if error
```

Comment:
The comment or the method's behavior needs to be updated. Currently the method returns 'no error' when the result is partial.